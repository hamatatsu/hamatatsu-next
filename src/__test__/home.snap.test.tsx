import Home from "@/app/(home)/page";
import { render } from "@testing-library/react";

describe("Home", () => {
  it("renders unchanged", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
