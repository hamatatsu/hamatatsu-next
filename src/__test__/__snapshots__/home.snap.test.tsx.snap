// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders unchanged 1`] = `
<div>
  <div
    class="root"
  >
    <div
      class="root"
    >
      <div
        class="avator"
      >
        <img
          alt="a profile image of hamatatsu"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
      <div>
        <h1
          class="title"
        >
          Welcome to my page
        </h1>
      </div>
      <div
        class="root"
      >
        <div
          class="socialButton"
        >
          <a
            class="socialButton"
            href="https://twitter.com/hamatatsu0"
          >
            <img
              alt="x"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
              srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
            />
          </a>
        </div>
        <div
          class="socialButton"
        >
          <a
            class="socialButton"
            href="https://github.com/hamatatsu"
          >
            <img
              alt="github"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
              srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="root"
    >
      <div
        class="copyright"
      >
        <p>
          Â© 
          2024
           hamatatsu
        </p>
      </div>
    </div>
  </div>
</div>
`;
